.github/workflows/docker.yml

```diff
@@ -1,7 +1,7 @@
 on:
   push:
     branches:
-    - main
+    - '*'
```

Comment:
This now runs on all PRs because the GHCR login and docker push steps are conditionally done only on main. This allows the docker build to gate merges, which is appropriate.